### PR TITLE
Security

### PR DESCRIPTION
**Pull request by IhorTarkhan**

### CODE REVIEW

**Comment by Natiik** [comment]
"customer" -> "client"

File: src/main/java/ua/knu/csc/iss/mvpbank/exceptions/BadRequestException.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package ua.knu.csc.iss.mvpbank.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.CONFLICT)
```
Comment (Natiik) [inline src/main/java/ua/knu/csc/iss/mvpbank/exceptions/BadRequestException.java]: Status change to BadRequest

File: src/main/java/ua/knu/csc/iss/mvpbank/security/filter/SuperAdminCustomerFilter.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package ua.knu.csc.iss.mvpbank.security.filter;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import ua.knu.csc.iss.mvpbank.entity.SuperAdmin;
+import ua.knu.csc.iss.mvpbank.repository.SuperAdminRepository;
+import ua.knu.csc.iss.mvpbank.security.JwtTokenProvider;
+
+@Component
+public class SuperAdminCustomerFilter extends AbstractSecurityFilter {
```
Comment (Natiik) [inline src/main/java/ua/knu/csc/iss/mvpbank/security/filter/SuperAdminCustomerFilter.java]: rename

**Comment by Natiik** [comment]
Businka

File: src/main/java/ua/knu/csc/iss/mvpbank/exceptions/BadRequestException.java (hunk 1 above)
Comment (IhorTarkhan) [reply src/main/java/ua/knu/csc/iss/mvpbank/exceptions/BadRequestException.java]: done

File: src/main/java/ua/knu/csc/iss/mvpbank/security/filter/SuperAdminCustomerFilter.java (hunk 2 above)
Comment (IhorTarkhan) [reply src/main/java/ua/knu/csc/iss/mvpbank/security/filter/SuperAdminCustomerFilter.java]: done

**Comment by IhorTarkhan** [comment]
changed customer to client

**Comment by IhorTarkhan** [comment]
zf `Businka` spapibo)))) ❤️

**Pull request merged**